libs/server.obj.php

```diff
@@ -512,6 +512,8 @@ public function jsonSerialize()
         return array(
                 'id' => $this->id,
                 'hostname' => $this->hostname,
+		'os' => $this->fk_os,
+		'pools' => $this->a_pool,
```

Comment:
the pools can not be filled with the array directly. You must loop through the pool and get pools to be an array. Filling this array with each pool's json output.

Something like:

``
$ret = array('id' => $this->id, ...........);
$ret['pools'] = array();
foreach($this->a_pool, as $pool) {
  $ret['pools'][] = $pool->jsonSerialize();
}
``

Of course, in the pool.obj.php, you need to define jsonSerialize() if it does not exist to build an array based off what info you want from the pool...

Hope it make sense!